src/components/cms/SkillVersion/SkillVersion.js

```diff
@@ -15,11 +15,11 @@ import TableBody from '@material-ui/core/TableBody';
 import TableCell from '@material-ui/core/TableCell';
 import TableHead from '@material-ui/core/TableHead';
 import TableRow from '@material-ui/core/TableRow';
-import _Button from '@material-ui/core/Button';
+import Button from '@material-ui/core/Button';
```

Comment:
Do not change this.

Reply:
There are 2 types of button one with style and one without. I have written the styled component for the styled button below. What other approach do you suggest?